sql/2024/04/image-formats.sql

```diff
@@ -0,0 +1,92 @@
+DECLARE
```

Comment:
```suggestion
# HTTP Archive query to image formats in WordPress.
#
# WPP Research, Copyright 2024 Google LLC
#
# Licensed under the Apache License, Version 2.0 (the "License");
# you may not use this file except in compliance with the License.
# You may obtain a copy of the License at
#
#     https://www.apache.org/licenses/LICENSE-2.0
#
# Unless required by applicable law or agreed to in writing, software
# distributed under the License is distributed on an "AS IS" BASIS,
# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
# See the License for the specific language governing permissions and
# limitations under the License.
#
# See https://github.com/GoogleChromeLabs/wpp-research/pull/97

DECLARE
```

Reply:
Thanks, I always accidentally remove the copyright header again when copying the code from the GCP console.